Source/WebCore/platform/graphics/skia/FontCascadeSkia.cpp

```diff
@@ -52,10 +52,13 @@ void FontCascade::drawGlyphs(GraphicsContext& graphicsContext, const Font& font,
     auto blob = builder.make();
     auto* canvas = graphicsContext.platformContext();
     auto* skiaGraphicsContext = static_cast<GraphicsContextSkia*>(&graphicsContext);
-    SkPaint paint = skiaGraphicsContext->createFillPaint();
+    SkPaint paint = graphicsContext.textDrawingMode() == TextDrawingMode::Fill ? skiaGraphicsContext->createFillPaint() : skiaGraphicsContext->createStrokePaint();
```

Comment:
I don't think this is correct. `TextDrawingModeFlags` is an OptionSet and both Fill and Stroke can be true. So, I think we should check first if fill is needed and do the fill and then do the stroke if present too.